docs/content/compress.md

```diff
@@ -117,11 +117,11 @@ Here are the advanced options specific to compress (Compress a remote).
 GZIP compression level (-2 to 9).
 
 			Generally -1 (default, equivalent to 5) is recommended.
-			Levels 1 to 9 increase compressiong at the cost of speed.. Going past 6 
+			Levels 1 to 9 increase compressiong at the cost of speed. Going past 6 
```

Comment:
This needs to be changed in compress.go instead. See marker on line 85:

```
{{< rem autogenerated options start" - DO NOT EDIT - instead edit fs.RegInfo in backend/compress/compress.go then run make backenddocs" >}}
```

Same with rest of the changes below this point.